### DISCUSSION TITLE
Grid Point Retrieval Intermittent Error

### DISCUSSION THREAD

**Discussion by grfritz** [weather-gov/api#381] [Q&A]
I have been initially trying out the API grid point forecast feature (i.e. https://api.weather.gov/gridpoints/IND/57,68) and am occasionally getting the following error, which can persist for 10 to 15 minutes:

"{ "correlationId": "b8a30cd9-f270-4d64-8d37-f44b35e5ee3c", "title": "Unavailable Resource", "type": "https://api.weather.gov/problems/UnavailableWebService", "status": 503, "detail": "The resource you requested is currently unavailable. Please try again later.", "instance": "https://api.weather.gov/requests/b8a30cd9-f270-4d64-8d37-f44b35e5ee3c" }.

I was preparing to initially employ this API function into our Skywarn service and begin phasing out our dependency on the NDFD but am now hesitating since the API does not appear fully stable.  Appreciate any feedback on getting this issue resolved.

Thanks.

**Comment by StephenClouse** [comment]
This should be reported to the TOC as an operational issue.
